lib/src/ultrahdr_api.cpp

```diff
@@ -26,6 +26,38 @@
 
 namespace ultrahdr {
 
+// Default configurations
+// Map is quarter res / sixteenth size
+const size_t kMapDimensionScaleFactorDefault = 4;
+
+// JPEG compress quality (0 ~ 100) for gain map
+const int kMapCompressQualityDefault = 85;
+
+// Gain map calculation
+const bool kUseMultiChannelGainMapDefault = false;
+// Default gamma value for gain map
+const float kGainMapGammaDefault = 1.0f;
+
+// The current JPEGR version that we encode to
+const char* const kGainMapVersion = "1.0";
+const char* const kJpegrVersion = "1.0";
```

Comment:
This is JPEG/R related, I prefer to keep it under jpegr.h

Reply:
Done